whipper/test/test_common_path.py

```diff
@@ -9,22 +9,28 @@
 class FilterTestCase(common.TestCase):
 
     def setUp(self):
-        self._filter = path.PathFilter(special=True)
+        self._filter_default = path.PathFilter()
+        self._filter_posix = path.PathFilter(posix=True, fat=False)
+        self._filter_fat = path.PathFilter(posix=False, fat=True)
+        self._filter_both = path.PathFilter(posix=True, fat=True)
 
-    def testSlash(self):
-        part = u'A Charm/A Blade'
-        self.assertEqual(self._filter.filter(part), u'A Charm-A Blade')
+    def testPosix(self):
+        part = u'A Charm/A \x00Blade'
+        self.assertEqual(self._filter_posix.filter(part), u'A Charm_A _Blade')
 
     def testFat(self):
         part = u'A Word: F**k you?'
-        self.assertEqual(self._filter.filter(part), u'A Word - F__k you_')
+        self.assertEqual(self._filter_fat.filter(part), u'A Word_ F__k you_')
 
-    def testSpecial(self):
-        part = u'<<< $&*!\' "()`{}[]spaceship>>>'
```

Comment:
This seems like a good string to test still. Any particular reason you're not including this anymore?